nix/tools.nix

```diff
@@ -183,6 +174,9 @@ in
   chktex = tex;
   commitizen = commitizen.overrideAttrs (_: _: { doCheck = false; });
   bats = if bats ? withLibraries then (bats.withLibraries (p: [ p.bats-support p.bats-assert p.bats-file ])) else bats;
+  git-annex = if stdenv.isDarwin then null else git-annex;
```

Comment:
Setting these to `null` is kinda sketchy (`meta.broken` would be better) but it's what the other derivations in this file do so I left it as-is.